File: Q14.c
import java.util.*;
import java.util.Scanner;
public class Q14{ 
    public static void main(String[] args) {
        Scanner scan=new Scanner(System.in);
        int baseSalary=scan.nextInt();
        int sessions=scan.nextInt();
        int payPerSession=scan.nextInt();
        int bonus=scan.nextInt();
        int maintenanceFee=scan.nextInt();
        int prod=sessions*payPerSession;
        int sum=prod+baseSalary+bonus-maintenanceFee;
        System.out.println(sum);
    }
}

File: Java while & Do while loops/Q14.java
import java.util.*;
import java.util.Scanner;
public class Q14{
    public static void main(String[]args){
        Scanner scan=new Scanner(System.in);
        int n=scan.nextInt();
        int temp=n;
        int bin=n;
        int c=0,dig=0,sum=0;
       while(bin>0){
           bin=bin/10;
           c++;
       }
       while(n>0){
           dig=n%10;
           sum=sum+(int)Math.pow(dig,c);
           n=n/10;
       }
       if(sum==temp){
           System.out.println(temp+" is an Armstrong number");
       }
       else{
          System.out.println(temp+" is not an Armstrong number");
       }
    }
}
